docs/WritingTests.asciidoc

```diff
@@ -51,13 +51,15 @@ Here is an example in Perl:
 
 [source,perl]
 -------------------------------------------------------------------
-use Mojo::Base "basetest";
+use Mojo::Base "basetest"; # See note <1>
+
```

Comment:
unrelated change. I would prefer to not add that blank line.

Reply:
Can we lint that?

Reply:
Not in this branch, just wondering since it's a very common point that comes up